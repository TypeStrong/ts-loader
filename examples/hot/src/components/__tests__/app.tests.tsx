import * as React from 'react';
import * as renderer from 'react-test-renderer';
import { App } from '../app';

describe('app', () => {
    it('component renders as expected', () => {
        const component = renderer.create(
            <App />
        );
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});