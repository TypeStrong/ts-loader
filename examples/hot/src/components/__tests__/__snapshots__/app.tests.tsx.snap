// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app component renders as expected 1`] = `
<div
  className="ui container"
>
  <h1>
    Heya, heya, heya!
  </h1>
</div>
`;
